### PR TITLE
Update RAG eval tool counter layout for easier interaction

### PR DESCRIPTION
**Pull request by shifucun**
image

### CODE REVIEW

File: static/css/eval_retrieval_generation.scss [hunk 1]
```diff
@@ -188,13 +189,12 @@ label {
 
   #form-container {
     border-top: solid 1px var(--gm-3-sys-light-outline-variant);
-    gap: 16px;
```
Comment (chejennifer) [inline static/css/eval_retrieval_generation.scss:191]: curious why you're removing this, how do the other feedback panes look with this change?

File: static/css/eval_retrieval_generation.scss [hunk 2]
```diff
@@ -188,13 +189,12 @@ label {
...
     padding: 16px 24px;
     flex-grow: 1;
     overflow: auto;
 
     .question-section .title {
-      font-size: 11px;
+      font-size: 16px;
```
Comment (chejennifer) [inline static/css/eval_retrieval_generation.scss:197]: why change the question title font size? these were specs from adriana's mocks

**Review by chejennifer — APPROVED** [review]
nice!

File: static/css/eval_retrieval_generation.scss (hunk 2 above)
Comment (shifucun) [reply static/css/eval_retrieval_generation.scss]: reverted

File: static/css/eval_retrieval_generation.scss (hunk 1 above)
Comment (shifucun) [reply static/css/eval_retrieval_generation.scss:191]: looks like this is block by some other css.

**Review by shifucun — COMMENTED** [review]
thanks for review! updated the heigh handling per offline discussion

**Pull request merged**